cdk/lib/recipes-backend.ts

```diff
@@ -335,6 +340,45 @@ export class RecipesBackend extends GuStack {
 			{ suffix: 'curation.json' },
 		);
 
+		new GuScheduledLambda(this, 'PublishContributors', {
+			app: 'recipes-publish-contributor-information',
+			architecture: Architecture.ARM_64,
+			fileName: 'profile-cache-rebuild.zip',
+			functionName: `PublishRecipeContributors-${this.stage}`,
+			handler: 'main.handler',
+			initialPolicy: [
+				new PolicyStatement({
+					effect: Effect.DENY,
+					actions: ['*'],
+					resources: [serving.staticBucket.bucketArn + '/content/*'],
+				}),
+				new PolicyStatement({
+					effect: Effect.ALLOW,
+					actions: ['s3:PutObject', 's3:GetObject'],
+					resources: [serving.staticBucket.bucketArn + '/*'],
+				}),
+			],
+			memorySize: 256,
+			monitoringConfiguration: {
+				noMonitoring: true, //TBD
+			},
+			rules: [
+				{
+					schedule: Schedule.cron({ minute: '16' }),
+					description:
+						'Update cache of contributor information for Feast at 16 minutes past every hour',
```

Comment:
ah interesting very specific time value 😁
i know, this is to get it done in calm time to avoid other process in between 👍

Reply:
Yeah try to avoid _everything_ hitting on the hour!